jboss-kie-process-migration/added/configuration/application-roles.properties

```diff
@@ -0,0 +1,2 @@
+kermit=admin,user
+gonzo=user
```

Comment:
Why include harcoded properties? Perhaps these would be better set as env vars, and then have the jboss-kie-process-migration.sh script dynamically create the users and roles properties files?

Reply:
+1 otherwise lgtm.